cloud/microsoft/office365/onedrive/mode/siteusage.pm

```diff
@@ -173,36 +177,45 @@ sub set_counters {
         },
     ];
     $self->{maps_counters}->{global} = [
-        { label => 'total-usage-active', set => {
+        { label => 'total-usage-active', nlabel => 'onedrive.sites.active.usage.total.bytes', set => {
                 key_values => [ { name => 'storage_used_active' } ],
                 output_template => 'Usage (active sites): %s %s',
                 output_change_bytes => 1,
                 perfdatas => [
-                    { label => 'storage_used_active', value => 'storage_used_active', template => '%d',
+                    { label => 'total_usage_active', value => 'storage_used_active', template => '%d',
                       min => 0, unit => 'B' },
                 ],
             }
         },
-        { label => 'total-usage-inactive', set => {
+        { label => 'total-usage-inactive', nlabel => 'onedrive.sites.inactive.usage.total.bytes', set => {
                 key_values => [ { name => 'storage_used_inactive' } ],
                 output_template => 'Usage (inactive sites): %s %s',
                 output_change_bytes => 1,
                 perfdatas => [
-                    { label => 'storage_used_inactive', value => 'storage_used_inactive', template => '%d',
+                    { label => 'total_usage_inactive', value => 'storage_used_inactive', template => '%d',
                       min => 0, unit => 'B' },
                 ],
             }
         },
-        { label => 'total-file-count', set => {
-                key_values => [ { name => 'file_count' } ],
+        { label => 'total-file-count-active', nlabel => 'onedrive.sites.active.files.total.count', set => {
+                key_values => [ { name => 'file_count_active' } ],
```

Comment:
Something's wrong between `file_count_active` and `active_file_count`.
They may be a "active" (for sites) missing somewhere.

Reply:
`onedrive.sites.active.files.total.count` is the number of files in active sites.
`onedrive.sites.files.active.total.count` is the number of active files (so in active sites).
We could rename the later into `onedrive.sites.active.files.active.total.count`, not sure though the additional `active` is needed here...
Up to you.
Same thing then for the last 3 global sharepoint metrics.